ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -101,119 +112,200 @@ extension QueueScheduler {
-	return _bridgedSignalProducer(from: signal, file: file, line: line).map { $0.map(bridgedTuple) }
+@available(*, unavailable, renamed:"SignalProducer(_:)")
+public func bridgedSignalProducer<Value>(from signal: RACSignal<Value>) -> SignalProducer<Value?, AnyError> {
+	fatalError()
 }
 
-/// Create a `SignalProducer` of 4-tuples which will subscribe to the provided
-/// signal once for each invocation of `start()`.
-///
-/// - parameters:
-///   - signal: The signal of `RACFourTuple` objects to bridge to a signal producer of 4-tuples.
-///   - file: Current file name.
-///   - line: Current line in file.
-///
-/// - returns: Signal producer created from the provided signal.
-public func bridgedSignalProducer<First, Second, Third, Fourth>(from signal: RACSignal<RACFourTuple<First, Second, Third, Fourth>>, file: String = #file, line: Int = #line) -> SignalProducer<(First?, Second?, Third?, Fourth?)?, AnyError> {
-	return _bridgedSignalProducer(from: signal, file: file, line: line).map { $0.map(bridgedTuple) }
+extension SignalProducer where Error == AnyError {
+	/// Create a `SignalProducer` which will subscribe to the provided signal once
+	/// for each invocation of `start()`.
+	///
+	/// - parameters:
+	///   - signal: The signal to bridge to a signal producer.
+	public init<SignalValue>(_ signal: RACSignal<SignalValue>) where Value == SignalValue? {
+		self.init { observer, disposable in
+			let failed: (_ error: Swift.Error?) -> () = { error in
+				observer.send(error: AnyError(error ?? defaultNSError("Nil RACSignal error")))
+			}
+
+			disposable += signal.subscribeNext(observer.send(value:),
+			                                   error: failed,
+			                                   completed: observer.sendCompleted)
+		}
+	}
 }
 
-/// Create a `SignalProducer` of 5-tuples which will subscribe to the provided
-/// signal once for each invocation of `start()`.
-///
-/// - parameters:
-///   - signal: The signal of `RACFiveTuple` objects to bridge to a signal producer of 5-tuples.
-///   - file: Current file name.
-///   - line: Current line in file.
-///
-/// - returns: Signal producer created from the provided signal.
-public func bridgedSignalProducer<First, Second, Third, Fourth, Fifth>(from signal: RACSignal<RACFiveTuple<First, Second, Third, Fourth, Fifth>>, file: String = #file, line: Int = #line) -> SignalProducer<(First?, Second?, Third?, Fourth?, Fifth?)?, AnyError> {
-	return _bridgedSignalProducer(from: signal, file: file, line: line).map { $0.map(bridgedTuple) }
+extension SignalProducer where Error == AnyError {
+	/// Create a `SignalProducer` of 1-tuples which will subscribe to the provided
+	/// signal once for each invocation of `start()`.
+	///
+	/// - parameters:
+	///   - signal: The signal of `RACOneTuple` objects to bridge to a signal producer of 1-tuples.
+	public init<First>(tupleSignal: RACSignal<RACOneTuple<First>>) where Value == First? {
+		self.init(tupleSignal, transform: bridgedTuple(from:))
```

Comment:
Well, I tried, but to no avail. It seems we need the label to disambiguate. `bridging:` is fine for me – I'll wait and see if someone else has an opinion about it and then change it.